src/Features/PaymentGatewaySuggestions/Init.php

```diff
@@ -106,26 +106,6 @@ public static function localize( $specs ) {
 			$data = (object) array_merge( (array) $locale, (array) $spec );
 			unset( $data->locales );
 
-			$data->fields = array();
```

Comment:
This code block is obsolete since we're fetching field information from the Payment Gateway API.